src/main/steps/applicant/yourhearings/hearings/content.ts

```diff
@@ -1,10 +1,30 @@
+import { applyParms } from '../../../../steps/common/url-parser';
 import { TranslationFn } from '../../../../app/controller/GetController';
 import { generateContent as yourhearingshearingscontent } from '../../../common/yourhearings/hearings/content';
+import { APPLICANT_TASK_LIST_URL, FETCH_CASE_DETAILS } from '../../../../steps/urls';
+import { CaseType, PartyType } from '../../../../app/case/definition';
 
 export { form } from '../../../common/yourhearings/hearings/content';
 export const generateContent: TranslationFn = content => {
   const hearingsContent = yourhearingshearingscontent(content);
+  const request = content.additionalData?.req;
+  const caseData = request.session.userCase;
+  if (content.additionalData?.req.session.userCase.caseTypeOfApplication === 'C100') {
+    hearingsContent.linkforsupport = '/applicant/hearing-needs/support-help';
```

Comment:
Don't hard code ur's here. Use enums for condition checking

Reply:
Yeah corrected this.